compiler/src/typed/typed_well_formedness.re

```diff
@@ -302,8 +302,12 @@ module WellFormednessArg: TypedtreeIter.IteratorArgument = {
           ],
         )
           when
-            modname == "Int32"
+            modname == "Int8"
```

Comment:
Should we apply these to `BigInt` and `Rational` as well?

Reply:
Yeah in that case I'm going to make an `isNumber` function

Reply:
Made that change